pytest_mypy.py

```diff
@@ -13,12 +13,20 @@ def pytest_addoption(parser):
     group.addoption(
         '--mypy', action='store_true',
         help='run mypy on .py files')
+    group.addoption(
+        '--mypy-ignore-missing-imports', action='store_true', 
+        help="suppresses error messages about imports that cannot be resolved ")
 
 
 def pytest_collect_file(path, parent):
     config = parent.config
+    mypy_config = []
+
+    if config.option.mypy_ignore_missing_imports:
+        mypy_config.append("--ignore-missing-imports")
+
     if config.option.mypy and path.ext == '.py':
```

Comment:
Maybe we should make it so the new option enables to plugin too?
That would allow
``` sh
pytest --mypy-ignore-missing-imports
```
instead of
``` sh
pytest --mypy --mypy-ignore-missing-imports
```

Reply:
Yeah, this is a good idea. I will fix that.

Thanks for comment :)

Reply:
I went ahead and did this in https://github.com/dbader/pytest-mypy/pull/17.

Thanks for your contribution!